tests/gdx-tests/src/com/badlogic/gdx/tests/extensions/ControllersTest.java

```diff
@@ -1,184 +1,283 @@
-/*******************************************************************************
```

Comment:
@davebaol The problem was that I committed CR-LF line endings, although I don't know how that happened as core.autocrlf is set to true (I just checked it). I fixed the line endings and now it should work.